app/templates/components/metrics-counter.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <h3 class="panel-title">
+      {{pluralize count label without-count=true}}
```

Comment:
don't you already pluralize in the component?

Reply:
This is the laBel pluralization